src/util/irep.cpp

```diff
@@ -502,8 +510,8 @@ int irept::compare(const irept &i) const
   if(r!=0)
     return r;
 
-  const subt::size_type size=get_sub().size(),
-        i_size=i.get_sub().size();
+  const auto size = get_sub().size(), i_size = i.get_sub().size();
+
```

Comment:
I believe this is an unrelated change. Let's try to keep changes to an absolute minimum.

Reply:
ok